eng/performance/scenarios.yml

```diff
@@ -22,6 +22,12 @@ jobs:
         displayName: '${{ parameters.osName }} ${{ parameters.osVersion }} ${{ parameters.architecture }} ${{ parameters.kind }} ${{ parameters.machinePool }}'
         timeoutInMinutes: 320
         variables:
+        - ${{ if ne(parameters.runEnvVars, '')}}:
+          - name: runEnvVarsParam
+            value: "--run-env-vars ${{ join(' ', parameters.runEnvVars)}}"
```

Comment:
Since both branches of the condition set the same variable name, you can just put the `if` around the `value: ` line rather than having to duplicate `name: runEnvVarsParam`